app/models/user.rb

```diff
@@ -239,6 +239,9 @@ def questions
     Node.questions.where(status: 1, uid: id)
   end
 
+  def followers_who_dont_follow_tags(tags)
```

Comment:
Hi, StellaMaris - do you think we could try to narrowly create this method in a unique PR first, test it with unit tests, separate from everything else? 

I love that you are thinking ahead to the complex interaction between these pieces of code -- that's great! But if we break this PR up into smaller parts, we can test them and we can evaluate their fitness in smaller pieces, which will be easier organizationally. Does that make sense?